.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```

Comment:
Looks like there are still some issues with missing line ends. Is your linter working??

Reply:
I guess it's not linting this because it's not a .js or .jsx file??